src/main/java/com/facebook/openwifirrm/modules/Modeler.java

```diff
@@ -94,6 +96,10 @@ public static class DataModel {
 			new ConcurrentHashMap<>();
 
 		/** List of latest state per device. */
+		public Map<String, List<State>> latestStates =
```

Comment:
Please update javadoc to distinguish `latestStates` from `latestState`

Reply:
Where to put the javadoc? I assumed the description in line 91 and line 102 are enough. Open to any suggestions